packages/backend/src/api/resources/Enums.ts

```diff
@@ -20,7 +22,7 @@ export type OAuthStrategy = `oauth_${OAuthProvider}`;
 
 export type OrganizationInvitationStatus = 'pending' | 'accepted' | 'revoked';
 
-export type OrganizationMembershipRole = 'basic_member' | 'guest_member' | 'admin';
+export type OrganizationMembershipRole = OrganizationCustomRoleKey;
```

Comment:
Please forgive my ignorance but would we not want to include the default roles 'org:member' | 'org:admin' here as well? We had a customer reach out to support regarding this TS error.
![Screenshot 2023-12-19 at 11 14 19 PM](https://github.com/clerk/javascript/assets/3329665/fac8f1e9-312b-4d63-8da9-8906ae018cae)